.golangci.reference.yml

```diff
@@ -68,17 +68,13 @@ run:
   # Allow multiple parallel golangci-lint instances running.
   # If false, golangci-lint acquires file lock on start.
   # Default: false
-  allow-parallel-runners: false
+  allow-parallel-runners: true
```

Comment:
Do you wanna keep this as `false` for default even though this change? I noticed #4100 but didn't have a good answer so didn't respond to it. Thinking since there are more minor breaking changes and the reference is changed.

Reply:
I already changed a lot of things, I will not change everything in this PR :smile: 
Here, it's just a wrong value in the reference.

Reply:
No and that's appreciated! Just didn't follow why the reference changes this value that seems to be untouched in the rest of the PR.

Reply:
I reverted this change, I will create another PR.